doc/programming_guide/install/windows.rst

```diff
@@ -76,3 +76,34 @@ If you happen to be using pipenv, then the appropriate command is:
 
 ``python3 -m pipenv install arcade``
 
+Install Arcade under WSL
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you need to use arcade on Ubuntu WSL, here are multiple guides depending on what you want.
+You can also explore running headless.
+
+TODO: ^^^ link to docs page, help pls
```

Comment:
What do you want for a link?

Reply:
Link "running headless" to [what corresponds to this page](https://api.arcade.academy/en/latest/advanced/headless.html), if you think it's a good idea.

Reply:
If you hit the 'edit' on that page, it will take you to the source. The source starts off with:
```

.. _headless:

Headless Arcade
===============
```

That first line will allow you to create a reference with `:ref:\`headless\``